ttnn/cpp/ttnn/operations/eltwise/binary_backward/binary_backward_pybind.hpp

```diff
@@ -717,31 +720,39 @@ void bind_binary_backward_assign(py::module& module, const binary_backward_opera
                const ttnn::Tensor& grad_tensor,
                const ttnn::Tensor& input_tensor,
                const std::optional<ttnn::MemoryConfig>& memory_config,
-               const uint8_t& queue_id) -> std::vector<ttnn::Tensor> {
-               return self(queue_id, grad_tensor, input_tensor, memory_config);
+               const std::optional<ttnn::Tensor>& input_grad,
+               const uint8_t& queue_id) -> std::vector<std::optional<ttnn::Tensor>> {
+               return self(queue_id, grad_tensor, input_tensor, memory_config, input_grad);
             },
             py::arg("grad_tensor"),
             py::arg("input_tensor"),
             py::kw_only(),
             py::arg("memory_config") = std::nullopt,
-            py::arg("queue_id") = 0},
+            py::arg("input_a_grad") = std::nullopt,
```

Comment:
Just `input_grad` is enough?

Reply:
there needs to be a consistency with both binary assign and unary assign

Reply:
for a unary only op, just input_grad is enough